dags/trigger_k8s_cronjob.py

```diff
@@ -17,6 +18,17 @@ def get_cronjob(cronjob_name, namespace, batchApi):
             "Exception when calling BatchV1Api->list_namespaced_cron_job: %s\n" % e)
     return False
 
+# Get the pod name for the newly created job
+def get_pod_name(namespace, pod_label_selector, core_v1):
+    try:
+        pods_list = core_v1.list_namespaced_pod(
+            namespace, label_selector=pod_label_selector, timeout_seconds=10)
+        pod_name = pods_list.items[0].metadata.name
+        return pod_name
+    except ApiException as e:
+        logging.critical(
+            "Exception when calling CoreV1Api->list_namespaced_pod: %s\n" % e)
```

Comment:
the exception should be propagated here, otherwise the `retry_with_backoff` will never know there was an exception - just that `None` is returned